app/Providers/AppServiceProvider.php

```diff
@@ -26,19 +27,15 @@ public function register()
      */
     public function boot()
     {
-        $locale = 'en';
+        $currentLocale = 'en';
 
         if ($this->app->environment() !== 'testing') {
             Event::subscribe(SlackSubscriber::class);
             if ( ! $this->app->runningInConsole()) {
-                $locale = locale();
+                $currentLocale = locale();
             }
         }
 
-        $this->app->setlocale($locale);
-
-        View::composer('*', function ($view) use ($locale) {
-            $view->with('locale', $locale);
-        });
+        $this->app->setlocale($currentLocale);
```

Comment:
I wonder if this would be better:

```php
if ($this->app->environment() !== 'testing') {
    Event::subscribe(SlackSubscriber::class);
}

$this->app->setLocale($this->app->runningInConsole() ? 'en' : locale());
```
```

Reply:
Done.